README.md

```diff
@@ -9,7 +9,7 @@ designed with unique features that help with more specific use cases.
 
```

Comment:
Thanks for looking at the doc as a whole, and I agree this could use improvement. However, the scope of this PR is just adding the "Using Pebble" documentation, so I'm going to leave this for another PR.